zygrader/user.py

```diff
@@ -139,8 +78,8 @@ def logout():
     window = ui.get_window()
```

Comment:
can't get at the docstring for a comment, but this is close enough. the docstring needs updated